listenbrainz/webserver/static/js/src/personal-recommendations/SearchDropDown.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import { faUser } from "@fortawesome/free-solid-svg-icons";
+import ListenControl from "../listens/ListenControl";
+
+export type SearchDropDownProps = {
+  action: (event: string) => void;
+  suggestions?: Array<string> | null;
```

Comment:
Same comment as on the other file, I don't think the prop can be `null` so this should be:
```suggestion
  suggestions?: Array<string>;
```

The `?` already marks it as optional, which translates to `Array<string> | undefined`.


With that change, the `!` on line 14 should be removed.